src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -268,3 +268,46 @@ message ContainerHealthProvider {
     /// (Optional) Attributes to be used by the provider.
     map<string, string> attributes = 3;
 }
+
+/// An IP address.
+message Address {
```

Comment:
I believe we need only `AddressLocation` and `AddressAllocation` messages. `Address` is just a string value. `SignedAddressAllocation` adds only signature which could be part of `AddressAllocation`, and if not set be an empty array.

Reply:
These messages were introduced by @sargun as part of https://github.com/Netflix/titus-api-definitions/pull/94. These messages are used by the VPC service as well and I'll let him chime in on if these message could/should change. The main addition in this PR (103) is simply the introduction of the `SignedAddressAllocation` to `ContainerResources`.